turingschool-examples/flashcards-starter
# Refactor - Test Files

#### What's this PR do?
Refactors all test files to include:
- beforeEach at the beginning of the file to cut down on test length and readability
- replace `function()` with arrow functions `() =>`
- rewrite card instances as unique cards separate from information stored in the `data.js` file

#### Where should the reviewer start?
`Card-test.js`, `Deck-test.js`, `Turn-test.js` and `Round-test.js`.

#### How should this be manually tested?
Read through each test file and note the changes. Run the tests in the terminal using `npm test` to ensure that all tests pass.

#### Any relevant tickets?
This PR completes an earlier ticket referencing that refactoring would be completed.
